### PR TITLE
fix(onetrust-cmp): load script await

### PR DESCRIPTION
**Pull request by mmatsuk-sc**
## Description

[Monday ticket](https://applicaster.monday.com/boards/1615228456/pulses/3323190971?userId=23068084)

Fix case, when cmp modal init run before script is loaded

<details>
  <summary>Platforms Affected</summary>
  
- [ ] Android
- [ ] Android TV
- [ ] Apple TV (TVOS)
- [ ] Fire TV (FireOS)
- [ ] iOS
- [ ] LG TV (WebOS)
- [ ] Samsung TV (Tizen)
- [ ] Web
</details>

## Known or Pending Issues

`fill in this section with potential issues / limitations the reviewer(s) should know about`

## Screenshots

`If there is some kind of ui change please share screenshots`

## Version Info

`Does this update require a certain version of quickbrick, or does it rely on another pr to work?`

### CODE REVIEW

**Comment by mmatsuk-sc** [comment]
Please, don't merge it, found issue

**Comment by mmatsuk-sc** [comment]
> Please, don't merge it, found issue

resolved

File: plugins/applicaster-cmp-onetrust/src/index.js [hunk 1]
```diff
@@ -194,6 +185,22 @@ export const NativeScreen = (props: Props) => {
     };
   }, []);
 
+  React.useEffect(() => {
+    if (showWebScreen) {
+      const intervalID = setInterval(() => {
+        //render OneTrust CMP only in case script was loaded successfully
+        if (window.oneTrustTV) {
+          console.log('Script was loaded successfully');
```
Comment (Pretzi) [inline plugins/applicaster-cmp-onetrust/src/index.js:193]: @mmatsuk-sc can we replace the log with xray log?

Comment (mmatsuk-sc) [reply plugins/applicaster-cmp-onetrust/src/index.js]: @Pretzi done

**Pull request merged**